src/controllers/index.ts

```diff
@@ -7,6 +7,8 @@ export * as beneficialOwnerStatements from "./beneficial.owner.statements.contro
 export * as beneficialOwnerType from "./beneficial.owner.type.controller";
 export * as checkYourAnswers from "./check.your.answers.controller";
 export * as landing from "./landing.controller";
+export * as updateLanding from "./update.landing.controller";
```

Comment:
The update controllers should be in sub dir. Same for view.

Reply:
done now!